packages/ui-kit/src/hooks/useCounter/CounterMetric.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, RelativeTimeRange, useStorybookAccessToken, quotedStringRegex } from '../../helpers'
+import { useCounter } from './useCounter'
+
+const args = {
+  id: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: { id: process.env.STORYBOOK_METRIC_ID },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 30
+    }
+  },
+  name: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: { name: process.env.STORYBOOK_METRIC_UNIQUE_NAME_1 },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  },
+  custom: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: {
+      custom: {
+        dataPool: {
+          id: process.env.STORYBOOK_DATAPOOL_ID
+        },
+        expression: 'COUNT_DISTINCT(taco_total_price)'
+      }
+    },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  },
+  sum: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: {
+      sum: {
+        dataPool: {
+          id: process.env.STORYBOOK_DATAPOOL_ID
+        },
+        measure: {
+          columnName: 'quantity'
+        }
+      }
+    },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  }
+}
+
+const meta: Meta = {
+  title: 'Components/Metric',
+  tags: ['pattern'],
+  parameters: {
+    imports: ['useCounter', 'useAccessToken'],
+    isFunction: true,
+    codeTemplate: storybookCodeTemplate,
+    transformBody: (body: string) =>
+      body
+        .replace(
+          'function',
+          `      
+      function`
+        )
+        .replace(`useStorybookAccessToken(axiosInstance)`, 'useAccessToken()')
+        .replace(
+          `{
+      ...args.id,
+      accessToken
+    }`,
+          `${JSON.stringify(args.id)}`
+        )
+        .replace(
+          `{
+      ...args.name,
+      accessToken
+    }`,
+          `${JSON.stringify(args.name)}`
+        )
+        .replace(
+          `{
+      ...args.custom,
+      accessToken
+    }`,
+          `${JSON.stringify(args.custom)}`
+        )
+        .replace(
+          `{
+      ...args.sum,
+      accessToken
+    }`,
+          `${JSON.stringify(args.sum)}`
+        )
+        .replace(process.env.STORYBOOK_METRIC_ID, '<PROPELL_METRIC_ID>')
+        .replace(process.env.STORYBOOK_DATAPOOL_ID, '<PROPELL_DATAPOOL_ID>')
+        .replace(quotedStringRegex('LAST_N_DAYS'), 'RelativeTimeRange.LastNDays')
```

Comment:
I see Linter's complaints on my side.

<img width="926" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/9b43034c-f6f5-48c1-a38f-2b59350445ca">

Reply:
did you add that env variable to your local?
It looks good on my side
please check .env file

Reply:
Yes, I have checked it, and that's not the issue. The problem arises because the environment variables are of type `string | undefined`, while the replace function expects its first argument to be of type `string | RegExp`.
A quick fix would be as follows:
```
.replace(process.env.STORYBOOK_METRIC_ID ?? '', '<PROPELL_METRIC_ID>')
```

It's a valid ts error. Could you please try to restart TS server in VS Code?